src/js/services/characterService.spec.js

```diff
@@ -26,6 +26,39 @@ describe('Character Service', () => {
     });
   });
 
+  describe('getCharactersAll', () => {
+    // getCharactersAll was able to call the API, so the data came back fine.
+    describe('successful API call', () => {
+      const MOCK_CHARACTER_ARRAY = [
+        { name: 'Mock character one' },
+        { name: 'Mock character two' },
+      ];
+
+      it('returns an array of characters', async () => {
+        get.mockResolvedValueOnce(MOCK_CHARACTER_ARRAY);
+
+        const character = await getCharactersAll();
+        expect(character).toEqual(expect.arrayContaining(MOCK_CHARACTER_ARRAY));
+      });
+    });
+
+    // getCharactersAll wasn't able to access the API, so got an error message
+    describe('unsuccessful API call', () => {
+      let character;
+      const MOCK_ERROR = 'Unable to get all characters!';
```

Comment:
Nitpick ~ maybe think of updating ```Character``` to ```Characters``` ~ when naming variables try and think of the content that they hold. In this case, the value of ```Character``` is an array(list) of data pointing to a plural naming convention. 😇

#naming-is-hard

Reply:
That makes total sense. I updated it!